jobs/JGDAS_GLOBAL_AERO_ANALYSIS_FINALIZE

```diff
@@ -0,0 +1,64 @@
+#! /usr/bin/env bash
+
+source "${HOMEgfs}/ush/preamble.sh"
+export WIPE_DATA="NO"
+export DATA=${DATA:-${DATAROOT}/${RUN}aeroanl_${cyc}}
+source "${HOMEgfs}/ush/jjob_header.sh" -e "aeroanlfinal" -c "base aeroanl aeroanlfinal"
+
+##############################################
+# Set variables used in the script
+##############################################
+export CDATE=${CDATE:-${PDY}${cyc}}
+export CDUMP=${CDUMP:-${RUN:-"gfs"}}
+
+
+##############################################
+# Begin JOB SPECIFIC work
+##############################################
+
+GDATE=$(date +%Y%m%d%H -d "${CDATE:0:8} ${CDATE:8:2} - ${assim_freq} hours")
+export GDATE
+gPDY=${GDATE:0:8}
```

Comment:
Does this not need an `export`?

Reply:
it does to be usable by the python ex-script, I guess the question is 'is it used'?